flow/core/rewards.py

```diff
@@ -441,3 +441,29 @@ def miles_per_gallon(env, veh_ids=None, gain=.001):
     mpg /= 1609.0
 
     return mpg * gain
+
+def instantaneous_power(env, veh_id=None, gain=.001):
```

Comment:
with this changed to `veh_id` then the docstring and function are no longer valid. i think the plan was to just simplify this function for handling one vehicle at a time? i'm not 100% sure.